### PR TITLE
fix(docs-infra): a couple of minor fixes for angular.io

### PR DESCRIPTION
**Pull request by gkalpak**
☝️ (See individual commits for details.)

### CODE REVIEW

**Review by jessicajaniuk — APPROVED** [review]
LGTM :cookie:

reviewed-for: size-tracking

**Review by atscott — APPROVED** [review]
reviewed-for: size-tracking

File: aio/src/app/app.component.html [hunk 1]
```diff
@@ -28,13 +28,13 @@
...
-      <a href="https://twitter.com/angular" title="Twitter" aria-label="Angular on twitter">
+      <a mat-icon-button href="https://twitter.com/angular" title="Twitter" aria-label="Angular on twitter">
         <mat-icon svgIcon="logos:twitter"></mat-icon>
       </a>
-      <a href="https://github.com/angular/angular" title="GitHub" aria-label="Angular on github">
+      <a mat-icon-button href="https://github.com/angular/angular" title="GitHub" aria-label="Angular on github">
         <mat-icon svgIcon="logos:github"></mat-icon>
       </a>
-      <a href="https://youtube.com/angular" title="YouTube" aria-label="Angular on YouTube">
+      <a mat-icon-button href="https://youtube.com/angular" title="YouTube" aria-label="Angular on YouTube">
```
Comment (dario-piotrowicz) [inline aio/src/app/app.component.html:37]: the icons seem to have lost they hover effect (https://pr42584-566a40f.ngbuilds.io/.)

is this wanted?

Comment (dario-piotrowicz) [reply aio/src/app/app.component.html:37]: ah yeah I can see the hover effect removed from the `a`s below, sorry

File: goldens/size-tracking/aio-payloads.json [hunk 2]
```diff
@@ -3,7 +3,7 @@
     "master": {
       "uncompressed": {
         "runtime-es2017": 4619,
-        "main-es2017": 456795,
+        "main-es2017": 456863,
```
Comment (petebacondarwin) [inline goldens/size-tracking/aio-payloads.json]: Interestingly, while this change incurs a cost in JS, there is actually a decrease in the CSS that is downloaded (and must be parsed and applied by the browser). So it is not that clear what the net result is here.

I wonder if there may be value in also tracking the size of the `styles.css` file?

**Review by petebacondarwin — APPROVED** [review]
LGTM! Thanks for keeping things tight!

There is a typo in the last commit:

> This commit fixes the issue ~~be~~ _by_ also setting

**Review by petebacondarwin — APPROVED** [review]
Reviewed-for: size-tracking

File: goldens/size-tracking/aio-payloads.json (hunk 2 above)
Comment (gkalpak) [reply goldens/size-tracking/aio-payloads.json]: Yeah, I was thinking about that one too. I added a commit to start tracking the CSS size as well: https://github.com/angular/angular/pull/42584/commits/d05b95a17d18d3b3825d9ba672301ddc836e331e

**Review by petebacondarwin — APPROVED** [review]

**Pull request closed without merging**